pkg/cover/cover.go

```diff
@@ -48,3 +60,111 @@ func (cov Cover) Serialize() []uint32 {
 	}
 	return res
 }
+
+const EXT string = ".ko"
+
+func searchModuleName(data []byte) (string, error) {
+	key := []byte("name=")
```

Comment:
This can find "fooname=" as well.
We could search for "\x00name=", but it won't work for the first string in the section.
I think the simplest fix is:
```
data = append([]byte{0}, data...)
key := []byte("\x00name=")
```